src/components/Chart/LineChart.jsx

```diff
@@ -180,11 +182,27 @@ class LineChart extends Component {
     if (this.rect) {
       const { data } = this.props
       const minY = this.getDataMin()
-      this.areaGenerator.y0(() => this.y(minY.y))
+      this.areaGenerator.y0(() => {
+        let min = this.y(minY.y)
+
+        if (min === 0) {
+          min += this.getInnerHeight()
+        }
 
-      this.rect
-        .attr('width', this.getInnerWidth())
-        .attr('height', this.getInnerHeight())
+        if (!this.props.showAxis) {
+          min += this.props.margin.bottom
+        }
+
+        return min
+      })
+
+      let height = this.getInnerHeight()
```

Comment:
```
const innerHeight  = this.getInnerHeight()
const height = innerHeight + (this.props.showAxis ? 0 : this.props.marginBottom)
```
This lets us get rid of the let var.

innerHeight can be used directly in the y0 function.